docs/toast-uncontrolled.md

```diff
@@ -0,0 +1,70 @@
+---
+id: toastUncontrolled
+title: Uncontrolled
+---
+
+## Static (Uncontrolled Toast)
+
+For the static method to work the root component must be wrapped by `<Provider />`, since it needs rendered `<ToastContainer />` to work properly.
+
+- `showToast(params)` - show a Toast and then dismiss it after the specified duration has passed
+- `hideToast()` - dismiss the currently visible toast. Does nothing when there's no visible toast.
```

Comment:
Let's move this bottom of the page, shall we?

Don't forget to change both `showToast` and `hideToast` to use the same format as the params list.

Also, let's just remove the `(Uncontrolled Toast)`.